pubspec.yaml

```diff
@@ -45,8 +45,6 @@ dependencies:
   firebase_core: ^3.1.0
   firebase_messaging: ^15.0.1
   flutter_color_models: ^1.3.3+2
-  flutter_local_notifications: ^17.2.1
```

Comment:
Searching for remaining references to this package, I find just one:
```
$ git grep -i 'flutter.?local.?notif'
android/app/src/main/res/raw/keep.xml:     The issue is that the package:flutter_local_notifications API has us
```

So that should be updated too.